utils/land-patterns.stanza

```diff
@@ -2018,6 +2018,22 @@ public defn make-bga-landpattern (num-rows:Int, ; the number of rows of package
     val cy-sz = enlarge(pkg-sz, 2.0 * courtyard-excess)
     layer(Courtyard(Top)) = Rectangle(cy-sz)
 
+    ;-------------------------------------------------------------
+    ; Compute the orientation marker
+    val silk-solder-mask-clearance = clearance(current-rules(), MinSilkSolderMaskSpace)
+    val solder-mask-registration = clearance(current-rules(), SolderMaskRegistration)
+    val line-width = clearance(current-rules(), MinSilkscreenWidth)
+    val pad-loc = center(pad-locs[0])
+    val pol-x = x(pad-loc) - ball-diameter * 0.5 - solder-mask-registration
+    val pol-y = y(pad-loc) + ball-diameter * 0.5 + solder-mask-registration
+    val line-length = 3.0 * line-width
+    val dist  = solder-mask-registration + silk-solder-mask-clearance + line-width * 0.5 + 0.01
```

Comment:
I'm curious about the 3.0 and 0.01 constants.
Is the length being 3 times the width a standard or it just looks nice that way?
Not sure why the 0.01 is useful?

Reply:
They're magic numbers from @electricfungi

Reply:
🪄 💫 ⭐